diff --git a/CCDFDataModel/Makefile b/CCDFDataModel/Makefile
--- CCDFDataModel/Makefile
+++ CCDFDataModel/Makefile
@@ -1,4 +1,4 @@
-INCLUDEDIR=-I../hclasses  -I/usr/include/libxml2/ -I/usr/include/udunits2/ $(CPPFLAGS) $(LDFLAGS)
+INCLUDEDIR=-I../hclasses $(CPPFLAGS) $(LDFLAGS)
 
 CCOMPILER=g++ -Wall $(INCLUDEDIR) 
 
diff --git a/adagucserverEC/Makefile b/adagucserverEC/Makefile
--- adagucserverEC/Makefile
+++ adagucserverEC/Makefile
@@ -4,7 +4,7 @@
 CCDFDATAMODEL = ../CCDFDataModel
 CCDFDATAMODELOBJ = $(CCDFDATAMODEL)/CCDFDataModel.a
 
-INCLUDEDIR= $(CPPFLAGS) -I $(HELPERCLASSES) -I $(CCDFDATAMODEL) -I/usr/include/libxml2/  -I/usr/include/udunits2/ -I/usr/include/freetype2/ -I/usr/include/postgresql/ -I/usr/include/cairo/ -I/usr/include/gdal/
+INCLUDEDIR= $(CPPFLAGS) -I $(HELPERCLASSES) -I $(CCDFDATAMODEL) 
 
 
 
diff --git a/hclasses/Makefile b/hclasses/Makefile
--- hclasses/Makefile
+++ hclasses/Makefile
@@ -1,4 +1,4 @@
-INCLUDEDIR=. -I/usr/include/libxml2/ $(CPPFLAGS) $(LDFLAGS)
+INCLUDEDIR=. $(CPPFLAGS) $(LDFLAGS)
 
 
 CCOMPILER=g++ -Wall -I $(INCLUDEDIR)

diff --git a/adagucserverEC/CCairoPlotter.h b/adagucserverEC/CCairoPlotter.h
--- adagucserverEC/CCairoPlotter.h
+++ adagucserverEC/CCairoPlotter.h
@@ -39,10 +39,10 @@
 #include "CDebugger.h"
 #include "CTypes.h"
   #include <ft2build.h>
-  #include <freetype/freetype.h>
-  #include <freetype/ftglyph.h>
-  #include <freetype/ftoutln.h>
-  #include <freetype/fttrigon.h>
+  #include <freetype2/freetype.h>
+  #include <freetype2/ftglyph.h>
+  #include <freetype2/ftoutln.h>
+  #include <freetype2/fttrigon.h>
   #include FT_FREETYPE_H
 #include <stdio.h>
 #include <math.h>